File: dev/requirements/screen/cases/joke-optional-text.3.4.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 3.4 — the alternate tags under the body, bordered and secondary to the joke.
let jokeOptionalText_3_4 = RequirementCase(id: "3.4") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "joke-optional-text", id: "3.4")
    }
}
#endif

File: dev/requirements/screen/cases/joke-optional-text.3.4.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 3.4 — The joke's optional text renders under the body as bordered tags.
final jokeOptionalText_3_4 = RequirementCase('3.4', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.jokeTags,
    slug: 'joke-optional-text',
    id: '3.4',
  );
});
